generators/generator-base-private.js

```diff
@@ -1577,6 +1577,16 @@ module.exports = class JHipsterBasePrivateGenerator extends Generator {
       return;
     }
 
+    const definition = this.readEntityJson('User');
+    if (definition) {
+      if (definition.relationships && definition.relationships.length > 0) {
+        this.warning('Relationships on the User entity side will be disregarded');
+      }
+      if (definition.fields && (definition.fields.length > 1 || definition.fields[0].fieldName !== 'id')) {
+        this.warning('Fields on the User entity side will be disregarded');
```

Comment:
I'm not sure I get the warning based on the condition

Reply:
The warning condition should be more clear now.